pertpy/tools/_perturbation_space/_simple.py

```diff
@@ -168,6 +175,48 @@ def compute(
 
         return ps_adata
 
+    @_doc_params(common_plot_args=doc_common_plot_args)
+    def plot_psbulk_samples(
+        self,
+        adata: AnnData,
+        groupby: str,
+        show: bool = True,
+        return_fig: bool = False,
+        **kwargs,
+    ) -> Figure | None:
+        """Plot the pseudobulk samples of an AnnData object.
+
+        Plot the count number vs. the number of cells per pseudobulk sample.
+
+        Args:
+            adata: Anndata containing pseudobulk samples.
+            groupby: `.obs` column to color the samples by.
+            {common_plot_args}
+            **kwargs: Are passed to decoupler's plot_psbulk_samples.
+
+        Returns:
+            If `return_fig` is `True`, returns the figure, otherwise `None`.
+
+        Examples:
+            >>> import pertpy as pt
+            >>> adata = pt.dt.zhang_2021()
+            >>> ps = pt.tl.PseudobulkSpace()
+            >>> pdata = ps.compute(
+            ...     adata, target_col="Patient", groups_col="Cluster", mode="sum", min_cells=10, min_counts=1000
+            ... )
+            >>> ps.plot_psbulk_samples(pdata, groupby=["Patient", "Major celltype"], figsize=(12, 4))
+
+        Preview:
+            .. image:: /_static/docstring_previews/pseudobulk_samples.png
```

Comment:
1) How does this work? Does this require a sphinx plugin? ... looks more elegant than what I'm doing in scirpy
2) I couldn't find this docstring rendered in the readthedocs preview in the `CentroidSpace` API doc.